.github/workflows/nightly_build.yml

```
@@ -390,3 +392,7 @@ jobs:
               }
             ]
           }' ${{ secrets.LANGFLOW_ENG_SLACK_WEBHOOK_URL }}
+
+        env:
+          LANGFLOW_ENG_SLACK_WEBHOOK_URL: ${{ secrets.LANGFLOW_ENG_SLACK_WEBHOOK_URL }}
+          LANGFLOW_ENG_SLACK_WEBHOOK_URL: ${{ secrets.LANGFLOW_ENG_SLACK_WEBHOOK_URL }}
```

actionlint (1.7.11)

[error] key "LANGFLOW_ENG_SLACK_WEBHOOK_URL" is duplicated in "env" section. previously defined at line:397,col:11. note that this key is case insensitive

(syntax-check)